### ISSUE TITLE
Why there is java script executer used ?

### ISSUE THREAD

**Issue by beunas** [roniyast/obSeleniumBasics#1]
Why there is java script executer used ?

 //JavascriptExecutor js = (JavascriptExecutor)driver;
        //js.executeScript("arguments[0].click()", promptButton);

**Comment by roniyast** [comment]
Yes ma’am , While doing that prompt example an exception named ElementClickInterceptedException Occured because of a google ad overlapping and it was not allowing the button click .Inorder to solve that issue google suggested this one Java script executer.
